h2o-algos/src/main/java/hex/schemas/Word2VecSynonymsV3.java

```diff
@@ -0,0 +1,19 @@
+package hex.schemas;
+
+import water.Iced;
+import water.api.API;
+import water.api.schemas3.KeyV3;
+import water.api.schemas3.SchemaV3;
+
+public class Word2VecSynonymsV3 extends SchemaV3<Iced, Word2VecSynonymsV3> {
+  @API(help="Source word2vec Model", required = true, direction = API.Direction.INPUT)
+  public KeyV3.ModelKeyV3 model;
+  @API(help="Target word", required = true, direction = API.Direction.INPUT)
+  public String target;
```

Comment:
Minor: `target` is confusing name, word would be beet.

btw: why a single word is passed as an input? Why not array of words?